apps/game/src/sandbox/time_warp.rs

```diff
@@ -45,15 +48,20 @@ impl JumpToTime {
             Widget::col(vec![
                 Line("Jump to what time?").small_heading().into_widget(ctx),
                 if app.has_prebaked().is_some() {
-                    GeomBatch::from(vec![(
-                        ctx.style().icon_fg.alpha(0.7),
-                        area_under_curve(
-                            app.prebaked().active_agents(end_of_day),
-                            slider_width,
-                            50.0,
-                        ),
-                    )])
-                    .into_widget(ctx)
+                    if let Ok(polygon) = area_under_curve(
+                        app.prebaked().active_agents(end_of_day),
+                        slider_width,
+                        50.0,
+                    ) {
+                        GeomBatch::from(vec![(
+                            ctx.style().icon_fg.alpha(0.7),
+                            polygon,)])
+                        .into_widget(ctx)
+                    } else {
```

Comment:
How about `match area_under_curve(...)` instead, so in the error case, we can say something like `Err(err) => warn!("Not drawing area under curve: {err}")` and seeing the particular error? In case the thing that breaks one day changes, it leaves the error message specifics for elsewhere

Reply:
ahh of course! we should definitely be printing out the error